apiary-receiver/apiary-receiver-sqs/src/main/java/com/expedia/apiary/extensions/receiver/sqs/messaging/SqsMessage.java

```diff
@@ -0,0 +1,110 @@
+/**
```

Comment:
Rename this to `ApiarySqsMessage` as this is not any generic SqsMessage but a SqsMessage which has come from Apiary SNS Topic.